beacon-chain/core/epoch/epoch_processing_test.go

```diff
@@ -77,6 +76,43 @@ func TestUnslashedAttestingIndices_CanSortAndFilter(t *testing.T) {
 	}
 }
 
+func TestUnslashedAttestingIndices_DuplicatedAttestations(t *testing.T) {
+	// Generate 5 of the same attestations.
+	atts := make([]*pb.PendingAttestation, 5)
+	for i := 0; i < len(atts); i++ {
+		atts[i] = &pb.PendingAttestation{
+			Data: &ethpb.AttestationData{Source: &ethpb.Checkpoint{},
+				Target:    &ethpb.Checkpoint{Epoch: 0},
+				Crosslink: &ethpb.Crosslink{},
+			},
+			AggregationBits: bitfield.Bitlist{0xC0, 0xC0, 0x01},
+		}
+	}
+
+	// Generate validators and state for the 5 attestations.
+	validators := make([]*ethpb.Validator, params.BeaconConfig().MinGenesisActiveValidatorCount/16)
```

Comment:
Why /16? Is this assuming a certain value for `params.BeaconConfig().MinGenesisActiveValidatorCount`? What if that value changes, will it break this test?

Reply:
This was from `TestUnslashedAttestingIndices_CanSortAndFilter`. Given the bitfields `0xC0, 0xC0, 0x01`, `MinGenesisActiveValidatorCount/16` will satisfy that length of bitfields on per slot basis.  We can change it to /8, /36 but will need to update the bitfields